src/topics/resolver/resolver.api.ts

```diff
@@ -0,0 +1,32 @@
+import { FastifyRequest } from "fastify";
+import { GlobalResolver } from "./global-resolver";
+import { resolverRoutesSchemas } from "./resolver.api.schema";
+import { Api } from "api";
+import { FetchedData } from "topics/group";
+
+const routes = async (api: Api) => {
+  api.post(
+    "/resolver", 
+    { schema: resolverRoutesSchemas.resolveAll },
+    async (req) => {
+      const accounts: string[] = getAccounts(req);
+      const fetchedData: FetchedData = {};
+      for (const account of accounts) {
+        fetchedData[account] = 1;
+      }
+      const resolved = await new GlobalResolver().resolveAll(fetchedData);
```

Comment:
You need to use the service factory here

Reply:
fixed! thanks!